server/app/views/applicant/ApplicantBaseFragment.html

```diff
@@ -42,12 +42,7 @@
         th:class="'usa-step-indicator\_\_segment' +
                   ${i == applicationParams.blockIndex() ? ' usa-step-indicator\_\_segment--current' : ''} +
                   ${block.isCompletedInProgramWithoutErrors() ? ' usa-step-indicator\_\_segment--complete' : ''}"
-      >
-        <span
```

Comment:
(non-blocking) Does the span or the parent li need any extra aria values or is this enough for it to correctly interpret?

Reply:
good catch. I added an aria-hidden to the segments, as advised by uswds spec when there are no labels, and realized that the overall progress bar had an untranslated aria label so I added a message for that